src/server/utils/datacube-guard/datacube-guard.ts

```diff
@@ -16,17 +16,17 @@
 
 import { Request } from "express";
 import { DataCube } from "../../../common/models/data-cube/data-cube";
-import { DataCubeFixtures } from "../../../common/models/data-cube/data-cube.fixtures";
 
 export function checkAccess(dataCube: DataCube, req: Request) {
   var guard = dataCube && dataCube.cluster && dataCube.cluster.guardDataCubes || false;
+  const headers = req && req.headers || {};
```

Comment:
Is it possible for `req` to be falsy?

Reply:
Not really, this function will now accept headers, instead of full requests.